### DISCUSSION TITLE
直播全屏，底下黑边仍有礼物栏

### DISCUSSION THREAD

**Discussion by eve2ptp** [the1812/Bilibili-Evolved#4090] [综合讨论 / General]
如图
![snipaste_20230325_125742](https://user-images.githubusercontent.com/42791572/227697147-4adc2d1e-1015-4f3f-a3ca-d284aebe95df.png)

**Comment by Geeyun-JY3** [comment]
反馈 bug 去开 issue。
另外请善用搜索，重复：#4067
